content/chat/setup.textile

```diff
@@ -0,0 +1,102 @@
+
+|_. Feature |_. Capabilities |
+| Send and receive messages | Publish, Subscribe |
+| Message history | Subscribe, History |
+| Online status | Subscribe, Presence |
+| Room occupancy | Subscribe, Channel Metadata |
+| Typing indicators | Publish, Subscribe, Presence |
+| Room reactions | Publish, Subscribe |
+
+When setting the capabilities for Chat, you need to apply them to either a wildcard resource, or a wildcard resource prefixed with the chat namespace, for example:
+
+* @[chat]*@ and @*@, or
+* @[*]*@
+
+h2(#install). Install
+
+The Chat SDK is built on top of the Ably Pub/Sub SDK and uses that to establish a connection with Ably.
+
+Both SDKs are available as NPM modules and via CDN.
+
+h3(#npm). NPM
+
+Install the Pub/Sub SDK and the Chat SDK:
+
+```[sh]
+npm install @ably/chat
+```
+
+Import the SDKs into your project:
+
+```[javascript]
+import * as Ably from 'ably';
+import ChatClient from '@ably/chat';
+```
+
+h3(#cdn). CDN
+
+Reference the Pub/Sub SDK and the Chat SDK within the @<head>@ of your HTML file:
+
+```[html]
+<script src="https://cdn.ably.com/lib/ably.min-1.js"></script>
+<script src="https://cdn.ably.com/chat/index.bundle.js"></script>
+```
+
+h2(#instantiate). Instantiate a client
+
+Instantiate a realtime client using the Pub/Sub SDK and pass the generated client into the Chat constructor:
+
+```[javascript]
+const ably = new Ably.Realtime({ key: '{{API_KEY}}', clientId: '<clientId>'});
+const chatClient = new ChatClient(ably, {logHandler: '', logLevel: '' });
+```
+
+A "@ClientOptions@":/api/realtime-sdk#client-options object may be passed to the Pub/Sub SDK instance to further customize the connection, however at a minimum you must set an API key and provide a @clientId@ to ensure that the client is "identified":/auth/identified-clients.
+
+Additional options can also be passed to the Chat client to customize the following properties:
+
+|_. Property |_. Description |
+| logHandler | The function to call for each line of "log output":#error. The default is @console.log@. |
+| logLevel | The verbosity of the "log output":#error. Options are; @silent@, @info@, @trace@, @warn@, @error@ or @debug@. |
+
+h2(#error). Error handling
+
+Set the @logHandler@ and @logLevel@ properties when "instantiating a client":#instantiate to configure your log handler:
+
+```[javascript]
+const ably = new Ably.Realtime({ key: '{{API_KEY}}', clientId: '<clientId>'});
+const chatClient = new ChatClient(ably, {logHandler: logWriteFunc, logLevel: 'debug' });
+```
+
+The @logHandler@ property is your own function that will be called for each line of log output generated by the Chat SDK.
+
+The @logLevel@ sets the verbosity of logs that will be output by the SDK. The following log levels are available to set:
+
+|_. Level |_. Description |
```

Comment:
Can we sort in this order?

https://github.com/ably/ably-chat-js/blob/e74bd6f64f893606296810a03179fcdcbcd1a568/src/logger.ts#L136